source/slang/slang-ir-legalize-varying-params.cpp

```diff
@@ -2259,6 +2259,13 @@ class LegalizeShaderEntryPointContext
                 varOffset += offsetAttr->getOffset();
                 builder.addSemanticDecoration(key, toSlice("_slang_attr"), (int)varOffset);
             }
+            else
+            {
```

Comment:
Somebody need to debug this and figure out why `offsetAttr` is null, because it isn't supposed to be so. The change here is a bandaid instead of a fix of the root cause.